Platon/backend/app/follow_system/views.py

```diff
@@ -1,10 +1,250 @@
+    SendFollowRequestsForm, AcceptFollowRequestsForm, RejectFollowRequestsForm
+from app.follow_system.forms import get_followings_parser, get_followers_parser, get_follow_requests_parser, send_follow_requests_parser, \
+    accept_follow_requests_parser, reject_follow_requests_parser
+from app.follow_system.models import Follow, FollowRequests
+from app.auth_system.views import login_required, generate_token
+
+import datetime
+from dateutil import parser
+from hashlib import sha256
+from functools import wraps
+
+
+# Private Account Restrictions not implemented.
+class GetFollowingsAPI(Resource):
+
+    @api.doc(responses={200: 'Followings List is successfully returned',
+                        404: 'Followings List is empty',
+                        400: 'Input Format Error',
+                        500: 'Database Connection Error'})
+    @api.expect(get_followings_parser)
+    def post(self):
+        '''
+            Takes follower_id as input and returns the followings as a list.
+        '''
+        form = GetFollowingsForm(request.form)
+        if form.validate():
+            try:
+                followSearch = Follow.query.filter(Follow.follower_id == form.follower_id.data).all()
+            except:
+                return make_response(jsonify({'error': 'Database Connection Error'}), 500)
+
+            if followSearch is []:
+                return make_response(jsonify({'error': 'User not found'}), 404)
+
+            mylist = []
+            for follow in followSearch:
+                mylist.append(follow.following_id)
+
+            return make_response(jsonify(mylist), 200)
+
+        else:
+            return make_response(jsonify({'error': 'Input Format Error'}), 400)
+
+
+# Private Account Restrictions not implemented.
+class GetFollowersAPI(Resource):
+
+    @api.doc(responses={200: 'Followers List is successfully returned',
+                        404: 'Followers List is empty',
+                        400: 'Input Format Error',
+                        500: ' Database Connection Error'})
+    @api.expect(get_followers_parser)
+    def post(self):
+        '''
+            Takes following_id as input and returns the followers as a list.
+        '''
+        form = GetFollowersForm(request.form)
+        if form.validate():
+            try:
+                followSearch = Follow.query.filter(Follow.following_id == form.following_id.data).all()
+            except:
+                return make_response(jsonify({'error': 'Database Connection Error'}), 500)
+
+            if followSearch is []:
+                return make_response(jsonify({'error': 'User not found'}), 404)
+
+            mylist = []
+            for follow in followSearch:
+                mylist.append(follow.follower_id)
+
+            return make_response(jsonify(mylist), 200)
+
+        else:
+            return make_response(jsonify({'error': 'Input Format Error'}), 400)
```

Comment:
I also think that this can be a GET request also.  Also why login is not required to see this information?

Reply:
- GET is better, you're right. 
- I assumed guests can see the followings and followers lists of the public profiles. I've checked the requirements and couldn't find information regarding that. Also I think you're right. I decided to add login required property. Changes will be made during the day. Thanks.

Reply:
With the latest commit, I've applied the suggested changes.